modules/cms/classes/ComponentPartial.php

```diff
@@ -107,6 +107,30 @@ public static function loadOverrideCached($theme, $component, $fileName)
             $partial = Partial::loadCached($theme, $component->alias . '/' . $fileName);
         }
 
+        if ($partial === null) {
+            $partial = Partial::loadCached($theme, strtolower($component->name) . '/' . $fileName);
+        }
+
+        if ($partial === null) {
+            $partial = Partial::loadCached($theme, $component->name . '/' . $fileName);
+        }
```

Comment:
You don't need this section because it wasn't supported previously so there's no reason to support case sensitive names